client/app/pages/dashboards/DashboardPage.jsx

```diff
@@ -114,22 +120,31 @@ function DashboardComponent(props) {
   return (
     <div className="container" ref={setPageContainer} data-test={`DashboardId${dashboard.id}Container`}>
       <DashboardHeader
-        dashboardOptions={dashboardOptions}
+        dashboardConfiguration={dashboardConfiguration}
         headerExtra={
-          <DynamicComponent name="Dashboard.HeaderExtra" dashboard={dashboard} dashboardOptions={dashboardOptions} />
+          <DynamicComponent
+            name="Dashboard.HeaderExtra"
+            dashboard={dashboard}
+            dashboardConfiguration={dashboardConfiguration}
+          />
         }
       />
       {!isEmpty(globalParameters) && (
         <div className="dashboard-parameters m-b-10 p-15 bg-white tiled" data-test="DashboardParameters">
-          <Parameters parameters={globalParameters} onValuesChange={refreshDashboard} />
+          <Parameters
+            parameters={globalParameters}
+            onValuesChange={refreshDashboard}
+            sortable={canEditDashboard}
```

Comment:
This is supposed to be `editingDashboard` to make it only available when in edit mode.

Reply:
Initially I thought of having it available on view mode a nice to have feat, but I agree is something to be reserved to edit mode.